profile/README.md

```diff
@@ -10,6 +10,16 @@
   <br>
 </h1>
 
+<h3 align="center">
+  <br>
+  OUR MISSION & VISION
```

Comment:
add this below banners

Reply:
done cc @ramesht007

Reply:
it is not fixed yet, can try and remove this whole mission and vision part?

Reply:
done cc @ramesht007